packages/client/hmi-client/src/components/drilldown/tera-drilldown.vue

```diff
@@ -11,9 +11,12 @@
 			>
 			<main>
 				<template v-for="(tab, index) in tabs" :key="index">
-					<component :is="tab" class="drilldown-panel" v-show="selectedViewIndex === index" />
+					<component :is="tab" v-show="selectedViewIndex === index" />
 				</template>
 			</main>
+			<footer v-if="slots.footer">
+				<slot name="footer" />
+			</footer>
```

Comment:
In addition to the user having the ability to add a footer to the preview section I've decided to add a slot for if they want to add one for the entire drilldown.  This will be useful in cases where the footer is shared between panels and not just for each section:
<img width="1786" alt="Screenshot 2023-11-29 at 3 52 53 PM" src="https://github.com/DARPA-ASKEM/terarium/assets/33158416/fc4b634e-86b0-4d04-9a72-07812e55f583">